modules/button/css/lib/button.scss

```diff
@@ -49,32 +60,58 @@ $_wdc-btn-caret-down-icon-white: svg-load(
   &.wdc-btn-hover,
   &:hover {
     background-color: $wdc-btn-bg-secondary-hover;
+    @include wdc-icon-color(#333d47);
   }
 
   &.wdc-btn-active,
   &:active {
     background-color: $wdc-btn-bg-secondary-active;
+    @include wdc-icon-color(#333d47);
   }
 
   &:focus,
   &.wdc-btn-focus {
     background-color: $wdc-btn-bg-secondary-focus;
     box-shadow: $wdc-btn-focus-ring;
     outline: none;
+    @include wdc-icon-color(#7b858f);
+  }
+
+  &:focus:hover {
+    @include wdc-icon-color(#333d47);
   }
 
   &.wdc-btn-disabled,
   &:disabled {
     background-color: $wdc-btn-bg-secondary-disabled;
     color: $wdc-btn-secondary-disabled-text-color;
     cursor: default;
+    @include wdc-icon-color($wdc-btn-secondary-disabled-text-color);
   }
 
   &.wdc-btn-grow {
     width: 100%;
     max-width: 100%;
   }
 
+  .wdc-icon[data-category="system"]:not([data-circle-background]) {
+    vertical-align: -7px;
```

Comment:
What does this do? It doesn't seem to make a difference if I remove it.

Reply:
This is for the medium size

Reply:
Can you describe what it's actually doing though? Why is it needed?

Reply:
Here's how the medium primary button with icon looks like without this: 
![image](https://user-images.githubusercontent.com/723109/71357398-ed47b880-2585-11ea-8ac1-6018e88b1801.png)

Reply:
Cool, thank you! Just wanted to make sure it was necessary as I didn't notice this when I disabled it 👍